### DISCUSSION TITLE
Árucsere bizonylat megvalósítása

### DISCUSSION THREAD

**Discussion by AttilaPletyak** [nav-gov-hu/eRECEIPT#2] [Ötletek]
Bizonyos kereskedőknél gazdasági eseményként jelentkezik az, hogy a vásárló visszahoz egy terméket és helyette kap egy másikat. 
Ez a másik termék lehet árban kedvezőbb, árban azonos vagy árban drágább is. Jelenleg OPG-n visszáru + értékesítés műveletpárral lehet ezt megoldani, ami viszont extra papír fogyásnak minősül. 

Online számla rendszerben van lehetőség olyan számla helyesbítésre, ahol be lehet küldeni pozitív és negatív értékkel a kiadott és visszavett árut.

Szükséges lenne olyan eNyugta kiterjesztés is, ahol ezt egy dedikált bizonylat típussal tudná küldeni a kasszaprogram. Nem elég a nyugtán belül visszáru, vagy visszárun belüli eladás engedélyezése/kiterjesztése, mert extrém esetben megeshet, hogy eltérő ÁFA kategóriából kap a vásárló csere terméket. Ha ez dedikált bizonylattípus lenne, amit nem kötelező implementálni, akkor egy-egy üzemeltető vizsgálható lenne, hogy milyen sűrűn alkalmazza ezt a típust.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
